bot.py

```diff
@@ -75,29 +98,57 @@ async def _close(self, ctx, delay: typing.Optional[str]):
 
         await utils._close_thread(self.bot, ctx, self.modLogs)
 
-    @commands.has_any_role(config.modRole)
-    @commands.command(name='reply', aliases=['r'])
-    async def _reply_user(self, ctx, *, content: typing.Optional[str]):
+    @cog_ext.cog_slash(
+        name='reply',
+        guild_ids=guildList,
+        description='Replys to a modmail, with your username',
+        permissions={config.guild: [create_permission(config.modRole, SlashCommandPermissionType.ROLE, True)]},
```

Comment:
`{config.guild: [create_permission(config.modRole, SlashCommandPermissionType.ROLE, True)]}` should be made into a static variable and passed instead.